components/scream/src/physics/shoc/shoc_diag_second_moments_srf.cpp

```diff
@@ -0,0 +1,16 @@
+#include "shoc_diag_second_moments_srf_impl.hpp"
+#include "ekat/scream_types.hpp"
+
+namespace scream {
+namespace shoc {
+
+/*
+ *  * Explicit instantiation for doing ice melting on Reals using the
```

Comment:
Maybe we should remove the physics-specific words from these ETI comments, to make them easier to copy/paste